cmd/kubemove/main.go

```diff
@@ -9,11 +9,11 @@ func main() {
 	pflag.CommandLine = flags
 
 	/*
-	// TODO
-	root := plugin.NewCmdKubeMove()
-	if err := root.Execute(); err != nil {
-		os.Exit(1)
-	}
-	 */
+		// TODO
+		root := plugin.NewCmdKubeMove()
```

Comment:
Let's remove `cmd/kubemove` and also from .travis.yaml`.

Reply:
removed